client/src/components/PageLayout/Header.tsx

```diff
@@ -16,16 +15,26 @@ import NextLink from 'next/link';
 import { useRouter } from 'next/router';
 import React from 'react';
 
+import { ArrowDownIcon } from '@chakra-ui/icons';
```

Comment:
```suggestion
import { ChevronDownIcon } from '@chakra-ui/icons';
```
The chevron is a standard way of representing a collapsible menu, let's stick to UI elements people are familiar with.